app/Repositories/Activity/ActivityRepository.php

```diff
@@ -157,6 +157,14 @@ public function advanceSearchForm($data)
             $searchResultQuery = $searchResultQuery->postFilter('term', ['_index' => $data['model']]);
         }
 
+        if (isset($data['isPublic']) && is_bool($data['isPublic'])) {
+            $searchResultQuery = $searchResultQuery->isPublic($data['isPublic']);
+        }
+
+        if (isset($data['elasticsearch']) && is_bool($data['elasticsearch'])) {
```

Comment:
Same here.

Reply:
"is_bool" is needed here,
I cannot depend on person call this function to ensure that value is indeed boolean.